packages/eslint-plugin/src/utils/configHelpers.js

```diff
@@ -14,7 +14,7 @@ const testFiles = [
 
 const docsFiles = ['**/*Page.tsx', '**/{docs,demo}/**', '**/*.doc.{ts,tsx}'];
 
-const configFiles = ['./just.config.ts', './gulpfile.ts', './*.js', './.*.js', './config', './scripts', './tasks'];
+const configFiles = ['./just.config.ts', './gulpfile.ts', './*.js', './.*.js', './config/**', './scripts', './tasks'];
```

Comment:
Would `**` globs also be needed on `scripts` and `tasks` to make linting those work properly?